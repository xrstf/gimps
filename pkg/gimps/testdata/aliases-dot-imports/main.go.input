package main

import (
	"fmt"
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	"k8s.io/api/core/v1"
	"k8s.io/api/networking/v1beta1"
	"log"
)

func main() {
	Describe("test", func() {
		fmt.Println(v1.ConfigMap{})
		fmt.Println(v1beta1.HTTPIngressPath{})
		log.Println("foo")
		Expect(true).To(BeTrue())
	})
}
