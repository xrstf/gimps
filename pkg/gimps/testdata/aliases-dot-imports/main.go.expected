package main

import (
	"fmt"
	"log"

	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	corev1 "k8s.io/api/core/v1"
	networkingv1beta1 "k8s.io/api/networking/v1beta1"
)

func main() {
	Describe("test", func() {
		fmt.Println(corev1.ConfigMap{})
		fmt.Println(networkingv1beta1.HTTPIngressPath{})
		log.Println("foo")
		Expect(true).To(BeTrue())
	})
}
